client/packages/coldchain/src/Equipment/api/api.ts

```diff
@@ -83,12 +93,21 @@ export const getAssetQueries = (sdk: Sdk, storeId: string) => ({
 
       const items = result?.assets;
 
+      return items;
+    },
+    logs: async (assetId: string) => {
+      const filter = { assetId: { equalTo: assetId } };
+      const sort = { key: AssetLogSortFieldInput.LogDatetime, desc: true };
+      const result = await sdk.assetLogs({ filter, sort, storeId });
+
+      const items = result?.assetLogs;
+
       return items;
     },
   },
   insert: async (input: InsertAssetInput): Promise<string> => {
     const result = await sdk.insertAsset({
-      input,
+      input: { ...input, storeId },
```

Comment:
leftover task : the asset should be store specific, we'll do the multi store version on central.